app/test/upgrade_test.go

```diff
@@ -51,6 +51,12 @@ func TestAppUpgrades(t *testing.T) {
 			key:           string(icahosttypes.KeyHostEnabled),
 			expectedValue: "true",
 		},
+		{
+			module:        "ICA",
+			subspace:      icahosttypes.SubModuleName,
+			key:           string(icahosttypes.KeyAllowMessages),
+			expectedValue: "[\"/ibc.applications.transfer.v1.MsgTransfer\",\"/cosmos.bank.v1beta1.MsgSend\",\"/cosmos.staking.v1beta1.MsgDelegate\",\"/cosmos.staking.v1beta1.MsgBeginRedelegate\",\"/cosmos.staking.v1beta1.MsgUndelegate\",\"/cosmos.staking.v1beta1.MsgCancelUnbondingDelegation\",\"/cosmos.distribution.v1beta1.MsgSetWithdrawAddress\",\"/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward\",\"/cosmos.distribution.v1beta1.MsgFundCommunityPool\",\"/cosmos.gov.v1.MsgVote\",\"/cosmos.feegrant.v1beta1.MsgGrantAllowance\",\"/cosmos.feegrant.v1beta1.MsgRevokeAllowance\"]",
+		},
```

Comment:
🤦‍♂️ I should've added this test when creating the original PR b/c it would've caught the regression when we added the module manager and used the default ICA module instead of an ICA module with an overwritten genesis state